src/index.ts

```diff
@@ -598,13 +598,14 @@ function FlatpickrInstance(
       self.config.errorHandler(e);
     }
 
-    if (triggerChange && self.currentYear !== oldYear) {
+    if (triggerChange && !isTimeInput() && self.currentYear !== oldYear) {
```

Comment:
Added this to fix the unit tests in ADS, but also because it doesn't make sense to try render the months dropdown for a timeinput. We should not trigger an event also, since the year did not change.

I will open a PR for flatpickr and see if they approve that change.